cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -275,6 +277,38 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 	// dummy interface is appended to PrevResult for use during cleanup
 	result.Interfaces = append(result.Interfaces, dummyInterface)
 
+	if r.NetworkPolicyMode == "strict" {
```

Comment:
nit: let's either add a comment above this section or move it into a new function. We should explain what strict mode is and why CNI is waiting

Reply:
Yeah this was mainly for testing purpose, have few pending tasks for the PR..